sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -155,6 +156,21 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     );
   }
 
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary details.
+   * @returns Scholastic Standing Summary details.
+   */
+  @IsBCPublicInstitution()
```

Comment:
I believe we are missing the decorator ` @HasStudentDataAccess("studentId")` here